Source/OCMockTests/NSMethodSignatureOCMAdditionsTests.m

```diff
@@ -39,6 +39,7 @@ - (void)testDeterminesThatSpecialReturnIsNotNeededForNonStruct
 
 - (void)testDeterminesThatSpecialReturnIsNeededForLargeStruct
 {
+    XCTSkipIf(TARGET_CPU_ARM64 && TARGET_OS_MAC, @"Apple Silicon Macs always print is special struct return? NO in [NSMethodSignature debugDescription]");
```

Comment:
This seems like an unrelated change.

Reply:
I bundles few small changes into a single PR. Do you want a separate PR for these two?
BWT, instead of parsing debug description I think it is possible to use `_isHiddenStructRet` private API. (Though it gives the same result - always returns NO on M1)